engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java

```diff
@@ -21,6 +21,7 @@ private SortableFieldMetadata(String fieldName) {
 
 	/**
 	 * The name of the field whose field bridge will be used to determine the value for the doc value field.
+	 * @return the name of the field
```

Comment:
So the build fails if that's not there? I left it out on purpose, because it's not useful and just adds clutter IMHO. If possible, I'd prefer to avoid this sort of comments.